### ISSUE TITLE
Disable creation of certain Mongo indexes?

### ISSUE THREAD

**Issue by clarkritchie** [matteodem/meteor-easy-search#393]
Hi and thanks for this package!

It looks like this package automatically creates a bunch of Mongo indexes when the app starts up.

Is there any way to tell it to **not** create a specific index?  I'm occasionally getting a [MongoError: E11000 duplicate key error](https://github.com/okland/accounts-phone/issues/14) on a field that I don't ever need to search on.

I noticed I can:
* `db.users.getIndexes()` -- see the problem index
* `db.users.dropIndex('services.phone.verify.code_1');` -- index is gone
* restart app, index comes back
* `meteor remove easy:search` -- for testing
* restart app, no indexes
* `meteor add easy:search`
* restart app -- see the problem index

**Comment by matteodem** [comment]
Yes that might be the case, are you using the `MongoTextIndex` package? It creates a text index for the fields that you want to search on. You don't need to specify those fields as you could add a custom `selector` method to add other functionality in there.

Please show me an example of your configuration.

**Comment by clarkritchie** [comment]
No, I'm not using `MongoTextIndex`.  As above, when I add this package, I see the index in the db.  When I remove this package, the index goes away.

**Comment by matteodem** [comment]
You don't have any configuration at all? Simply adding the package creates indexes?

**Comment by clarkritchie** [comment]
Pretty much this:
```
// setup specific server-side EasySearch search indexes
if (Meteor.isServer) {
    // good configuration examples:  https://github.com/matteodem/easy-search-leaderboard/blob/master/leaderboard.js
    VouchersIndex = new EasySearch.Index({
        collection: Vouchers,
        defaultSearchOptions: {
            limit: 10000 // total WAG
        },
        fields: ['code', 'identifier'],
        engine: new EasySearch.MongoDB()
    });

    OrdersIndex = new EasySearch.Index({
        collection: Orders,
        defaultSearchOptions: {
            limit: 10000 // total WAG
        },
        fields: ['userId'],
        engine: new EasySearch.MongoDB()
    });

    PaymentsIndex = new EasySearch.Index({
        collection: Payments,
        defaultSearchOptions: {
            limit: 10000 // total WAG
        },
        fields: ['senderPhone', 'transactionReference', 'firstName', 'lastName'],
        engine: new EasySearch.MongoDB()
    });

    UsersIndex = new EasySearch.Index({
        collection: Meteor.users,
        defaultSearchOptions: {
            limit: 10000 // total WAG
        },
        fields: ['profile.name', 'phone.number'],
        engine: new EasySearch.MongoDB()
    });
}

```

**Comment by matteodem** [comment]
I'll lower the priority of this as the only place in the core code that touches / creates an index is when the `MongoTextIndex` engine is used. See [here](https://github.com/matteodem/meteor-easy-search/blob/master/packages/easysearch:core/lib/engines/mongo-text-index.js#L48)

**Comment by matteodem** [comment]
This seems to have created problems with other users too so I'l prioritize this now
